docker-compose.yml

```diff
@@ -43,6 +43,7 @@ services:
       - bredis_1
       - bredis_2
       - bconsul
+      - jaeger
```

Comment:
Call this `bjaeger` to keep the same style as the rest of the docker-compose file. As a result, each of the configs in `config-next` that you've added `jaeger:4317` will need to become `bjaeger:4317`.

Reply:
done